sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/xds/cache/XdsDataCache.java

```diff
@@ -54,7 +55,7 @@ public class XdsDataCache {
     /**
      * key:service name value:cluster map
      */
-    private static Map<String, List<String>> serviceNameMapping;
+    private static Map<String, List<String>> serviceNameMapping = new HashMap<>();
```

Comment:
Maybe add final and define as ConcurrentHashMap？

Reply:
update operation will replace the map.